licenses/InterMine

```diff
@@ -0,0 +1,452 @@
+InterMine is an open source project distributed under the GNU Lesser
+General Public Licence.
```

Comment:
This is LGPL-2.1. So no need to add this license, but simply reuse the on already there.